packages/progress-circle/src/ProgressCircle.ts

```diff
@@ -123,17 +143,17 @@ export class ProgressCircle extends SizedMixin(SpectrumElement, {
             if (
                 !this.label &&
                 !this.getAttribute('aria-label') &&
-                !this.getAttribute('aria-labelledby')
+                !this.getAttribute('aria-labelledby') &&
+                !this.slotHasContent
             ) {
                 window.__swc.warn(
                     this,
-                    '<sp-progress-circle> elements will not be accessible to screen readers without one of the following:',
+                    '<sp-progress-circle> elements will not be accessible to screen readers in the following situations:',
```

Comment:
@Westbrook I have modified the warning text to -> "<sp-progress-circle> elements need one of the following to be accessible:" because we have tests written specifically to check if the warning message has the word "accessible" in it. Is this good enough?

Reply:
Much better, thanks!